package.json

```diff
@@ -3,7 +3,6 @@
   "version": "1.0.0-semantically-released",
   "description": "Site footer component for use when building edX frontend applications",
   "main": "dist/main.js",
-  "module": "dist/main.js",
```

Comment:
Is this something we should look for in our other libraries?  Did you already do that?

Reply:
I'm going to revert this. I don't think it was involved in making the bundle smaller.